### ISSUE TITLE
Admin\Operations - Internal User can view an Operation

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1702]
#### Description:
As an internal user, I can see an Operation Information page

#### Acceptance Criteria:
Given I am on the [operations grid](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7861-688&t=0E4HRE1djEXL07dW-4)
When I click View Operation
Then I can see information about that operation
* Same as external operation information page, but in view only mode with no edit button
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=18140-836&t=0E4HRE1djEXL07dW-4)

The Issue BORO ID and BCGHGID buttons on the operation information page are being handled in ticket #2141 and #2152 

#### Development Checklist:

- [x] Modify page in folder  `bciers/apps/administration/app/idir/cas_admin/operations/[operationId]/page.tsx` to ` return   <OperationInformationPage operationId={operationId} />`  
- [x] Also(?), Modify page in folder `bciers/apps/administration/app/idir/cas_analyst/operations/[operationId]/page.tsx` to ` return   <OperationInformationPage operationId={operationId} />;`  
- [x] implement logic in `OperationInformationForm` for `SingleStepTaskListForm` property  `allowEdit={role === FrontEndRoles.INDUSTRY_USER_ADMIN}`
- [x] modify vitests for OperationInformationForm in view only mode with no edit button
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies
#1701 

- Blocked by
- Blocking
